test/components/views/context_menus/MessageContextMenu-test.tsx

```diff
@@ -52,17 +237,19 @@ function createMessageContextMenu(eventContent: ExtensibleEvent) {
         },
     );
 
-    const mxEvent = new MatrixEvent(eventContent.serialize());
     mxEvent.setStatus(EventStatus.SENT);
 
     client.getUserId = jest.fn().mockReturnValue("@user:example.com");
     client.getRoom = jest.fn().mockReturnValue(room);
 
     return mount(
-        <MessageContextMenu
-            chevronFace={null}
-            mxEvent={mxEvent}
-            onFinished={jest.fn(() => {})}
-        />,
+        <RoomContext.Provider value={context as unknown as any}>
```

Comment:
may as well cast to the correct type instead of `any` since you're already casting